test/backend/DataVaultTestSuite.scala

```diff
@@ -68,7 +68,7 @@ class DataVaultTestSuite extends PlaySpec {
         "returns failure" when {
           "requesting invalid range" in {
             val result = (vaultPath / dataKey)
-              .readBytes(Some(Range.Long(-10, 10, 1)))(globalExecutionContext)
+              .readBytes(Some(Range.Long(-5, -10, 1)))(globalExecutionContext)
```

Comment:
Negative starts for byte ranges are now actually allowed in GCS. So we need to test an *even more invalid* range to check that a failure is indeed returned (this test is intended to check that no exceptions bubble up but instead the tryo turns them into Box.Failures).